### PR TITLE
Updated Deploy on GCP Doc

### LINKED ISSUE #15969: Documentation: Rewrite the Airbyte GCP deployment guide
**Note:** If you'd like to work on this issue, please read the [Docs Contributor guide](https://docs.airbyte.com/contributing-to-airbyte/updating-documentation) and comment on the issue to confirm it is still up for grabs.

The current [Airbyte GCP deployment guide](https://docs.airbyte.com/deploying-airbyte/on-gcp-compute-engine) doesn't follow the [Google style guide](https://developers.google.com/style) and overuses screenshots instead of a well-documented procedure. 

Tasks:
- Document the procedure in words instead of screenshots (similar to the [Airbyte Cloud quickstart](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud))
- Ensure it adheres to the Google style guide

### PR DESCRIPTION
**Pull request by PragatiVerma18**
## What
*Updated the Deploy on GCP Doc to adhere to the Google style guide.* 
Fixes #15969

## How
- *Document the procedure in words instead of screenshots (similar to the [Airbyte Cloud quickstart](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud))*
- *Ensure it adheres to the Google style guide*

## 🚨 User Impact 🚨
There are no breaking changes.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@Amruta-Ranade can you check this contribution?

File: docs/deploying-airbyte/on-gcp-compute-engine.md [hunk 1]
```diff
@@ -91,59 +61,53 @@ sudo apt-get install -y docker-ce docker-ce-cli containerd.io
...
-* Create ssh tunnel.
+1. In your local terminal, create an SSH tunnel to connect the GCP instance to Airbyte
 
 ```bash
-# In your workstation terminal
 gcloud --project=$PROJECT_ID beta compute ssh $INSTANCE_NAME -- -L 8000:localhost:8000 -N -f
 ```
 
-* Just visit [http://localhost:8000](http://localhost:8000) in your browser and start moving some data!
+2. Verify the connection by visiting [http://localhost:8000](http://localhost:8000) in your browser
```
Comment (Amruta-Ranade) [inline docs/deploying-airbyte/on-gcp-compute-engine.md:109]: Add period (.) at the end of the sentence

**Review by Amruta-Ranade — CHANGES_REQUESTED** [review]
Added comments for formatting changes

File: docs/deploying-airbyte/on-gcp-compute-engine.md [hunk 2]
```diff
@@ -3,85 +3,55 @@ import TabItem from '@theme/TabItem';
...
-* Connect to your instance
+5. Connect to your instance in your local terminal.
 
 ```bash
-# In your workstation terminal
 gcloud --project=$PROJECT_ID beta compute ssh $INSTANCE_NAME
 ```
 
-* Install `docker`
+6. Install docker on your VM instance terminal by following the below commands in your ssh session on the instance terminal.
```
[the remark this answers is not in the document]
Comment (PragatiVerma18) [reply docs/deploying-airbyte/on-gcp-compute-engine.md]: Done

[the remark this answers is not in the document]
Comment (PragatiVerma18) [reply docs/deploying-airbyte/on-gcp-compute-engine.md]: Done

File: docs/deploying-airbyte/on-gcp-compute-engine.md (hunk 1 above)
Comment (PragatiVerma18) [reply docs/deploying-airbyte/on-gcp-compute-engine.md]: Done

File: docs/deploying-airbyte/on-gcp-compute-engine.md [hunk 3]
```diff
@@ -3,85 +3,55 @@ import TabItem from '@theme/TabItem';
...
 
 ```bash
-# In your workstation terminal
 PROJECT_ID=PROJECT_ID_WHERE_YOU_CREATED_YOUR_INSTANCE
-INSTANCE_NAME=airbyte # or anyother name that you've used
+INSTANCE_NAME=airbyte # or any other name that you've used
 ```
 
-* Install `gcloud`
+Follow the steps below if you’re working on macOS:
```
[the remark this answers is not in the document]
Comment (PragatiVerma18) [reply docs/deploying-airbyte/on-gcp-compute-engine.md]: Done

File: docs/deploying-airbyte/on-gcp-compute-engine.md [hunk 4]
```diff
@@ -3,85 +3,55 @@ import TabItem from '@theme/TabItem';
...
-:::info
-
-Note: The following commands will be entered either on your local terminal or in your ssh session on the instance terminal. The comments above each command block will indicate where to enter the commands.
-
-:::
+## Set up the environment
 
-* Set variables in your terminal
+1. Create a new GCP instance.
+2. Set variables in your local terminal
```
[the remark this answers is not in the document]
Comment (PragatiVerma18) [reply docs/deploying-airbyte/on-gcp-compute-engine.md]: Done

File: docs/deploying-airbyte/on-gcp-compute-engine.md [hunk 5]
```diff
@@ -3,85 +3,55 @@ import TabItem from '@theme/TabItem';
...
-
-:::info
-
-Note: The following commands will be entered either on your local terminal or in your ssh session on the instance terminal. The comments above each command block will indicate where to enter the commands.
-
-:::
+## Set up the environment
 
-* Set variables in your terminal
+1. Create a new GCP instance.
```
[the remark this answers is not in the document]
Comment (PragatiVerma18) [reply docs/deploying-airbyte/on-gcp-compute-engine.md]: Done

File: docs/deploying-airbyte/on-gcp-compute-engine.md [hunk 6]
```diff
@@ -91,59 +61,53 @@ sudo apt-get install -y docker-ce docker-ce-cli containerd.io
 sudo usermod -a -G docker $USER
 ```
 
-* Install `docker-compose`
+7. Install docker-compose on your VM instance terminal by following the below commands in your ssh session on the instance terminal.
```
[the remark this answers is not in the document]
Comment (PragatiVerma18) [reply docs/deploying-airbyte/on-gcp-compute-engine.md]: Done

**Comment by PragatiVerma18** [comment]
Hey @Amruta-Ranade, I have updated the doc with formatting changes as requested. Please check and let me know if there are any further changes required. Thank you.

**Review by Amruta-Ranade — APPROVED** [review]

**Pull request merged**